src/main/java/at/hannibal2/skyhanni/api/enoughupdates/ItemResolutionQuery.kt

```diff
@@ -102,14 +118,11 @@ class ItemResolutionQuery {
                 val unCleanItemDisplayName: String = EnoughUpdatesManager.getDisplayName(internalName)
                 var cleanItemDisplayName = unCleanItemDisplayName.removeColor()
                 if (cleanItemDisplayName.isEmpty()) continue
-                if (isPet) {
+                if (petPattern.matches(itemName)) {
                     if (!cleanItemDisplayName.contains("[Lvl {LVL}] ")) continue
                     cleanItemDisplayName = cleanItemDisplayName.replace("[Lvl {LVL}] ", "")
```

Comment:
<img width="924" height="540" alt="Image" src="https://github.com/user-attachments/assets/d1a39655-405e-4093-969d-632aa8cc243f" />

Better for sure - but there is still room to "combine" these two branches. The outputs of the matcher are not used anywhere besides that conditional, so they can be "conjoined", to avoid pattern-matching more than we need to.

Essentially, use the `matchMatcher` _as_ the if statement.
I can explain more in Discord, if that was nonsensical...